gce/prepull.yaml

```diff
@@ -14,122 +14,83 @@ spec:
       nodeSelector:
         kubernetes.io/os: windows
       initContainers:
-      - image: e2eteam/busybox:1.29
-        name: busybox1
-        command: ["cmd", "exit"]
-      - image: e2eteam/curl:1803
-        name: curl2
+      - image: e2eteam/agnhost:2.8
+        name: agnhost
         command: ["cmd", "exit"]
-      - image: e2eteam/java:openjdk-8-jre
-        name: java3
+      - image: e2eteam/agnhost:2.6
+        name: agnhostprivate
```

Comment:
We should probably do something about this. 2.6 is an older image, and most likely it has a different image base than 2.8 (MS updates the published container images ~monthly), so this would increase the pull time significantly. Most of the other images should have the same base, since they're built together.

Reply:
Agreed. Looks like AgnhostPrivate is used in just a couple places: https://github.com/kubernetes/kubernetes/search?q=AgnhostPrivate&unscoped_q=AgnhostPrivate. Any reason not to just bump https://github.com/kubernetes/kubernetes/blob/master/test/utils/image/manifest.go#L211 from 2.6 to 2.8?

Reply:
No reason, it's just that the image has to be built and published manually into the gcr.io authenticated registry, and it's extremely difficult to get someone to do it. Also, the agnhost image gets regular updates and bumps (a lot of them were through my PRs, like the Image Centralization PRs), so getting someone to push the image to the authenticated registry multiple times is even more difficult. :)